### PR TITLE
Icon refresh

### PR DESCRIPTION
**Pull request by omanikhi**

### CODE REVIEW

File: branding/style/branding.css [hunk 1]
```diff
@@ -233,6 +236,9 @@
 }
 
 /* edit contents */
+#activities #btn_editfile {
```
Comment (takesson) [inline branding/style/branding.css]: Hmm, I would like to hide them on "details / open" page but keep them in "edit" page. 

Can we distinguish on "column-activities"?

**Review by takesson — CHANGES_REQUESTED** [review]

File: branding/style/branding.css (hunk 1 above)
Comment (omanikhi) [reply branding/style/branding.css]: Sure we can. Done.

Comment (takesson) [reply branding/style/branding.css]: Can you make the same change on use_url and open?

Comment (omanikhi) [reply branding/style/branding.css]: Ah sorry missed the "them". Done.

**Review by takesson — APPROVED** [review]

**Pull request merged**